src/utils/uri.ts

```diff
@@ -0,0 +1,4 @@
+export const urlPropsToObject = (url: string): object =>
```

Comment:
Better use well-tested and popular [qs](https://github.com/ljharb/qs) module for parsing.

Reply:
I think it's not so complicated point to use one more dependency.

Reply:
I'd argue that a well-maintained module by @tj is almost always better than our in-house solution. But it's up to you in this case.